app/models/qdm/patient.rb

```diff
@@ -0,0 +1,143 @@
+class QDM::Patient
+  include Mongoid::Document
+  field :birth_datetime, type: DateTime
+  field :qdm_version, type: String, default: '5.3'
+  field :ethnicity, type: QDM::Code
+  field :race, type: QDM::Code
+  field :sex, type: QDM::Code
+
+  # These are the "data criteria", or QDM datatype elements that exist on a
```

Comment:
Are these `data_criteria` analogs, or `entry` analogs?

Reply:
`entry` analogs, I think I used `data_criteria` term that is what is coming from the HQMF parser (the draggable elements in the Bonnie patient builder are called `data_criteria`).